src/shared/containers/GetOrganizations.tsx

```diff
@@ -35,108 +35,89 @@ import {convertStringToEpoch} from 'src/shared/utils/dateTimeUtils'
 import {updateReportingContext} from 'src/cloud/utils/reporting'
 
 // Types
-import {Me} from 'src/client/unityRoutes'
 import {PROJECT_NAME} from 'src/flows'
 import {RemoteDataState} from 'src/types'
 
 // Thunks
 import {getQuartzIdentityThunk} from 'src/identity/actions/thunks'
 
-const canAccessCheckout = (me: Me): boolean => {
-  if (Boolean(me?.isRegionBeta)) {
-    return false
-  }
-  return me?.accountType !== 'pay_as_you_go' && me?.accountType !== 'contract'
-}
-
```

Comment:
The purpose of this function was to convert the combination of the `isRegionBeta` and `accountType` properties into a `boolean` of whether the user could access checkout. We don't need that anymore, as `isUpgradeable` is a `boolean` that does just that.